src/Castle.Core.Tests/log4netIntegration/Log4netFactoryTestCase.cs

```diff
@@ -31,6 +29,7 @@ namespace Castle.log4netIntegration
 	[TestFixture]
 	public class Log4netFactoryTestCase
 	{
+		private const string log4netXmlPath = "./log4netIntegration/log4net.xml";
```

Comment:
Will do. Posting to follow up commits for this, we can squish everything down once you are ready to merge.